### PR TITLE
Dev avi nicobar

### PR DESCRIPTION
**Pull request by avinashpatel57**

### CODE REVIEW

File: backend/istyleyouapi/common.php [hunk 1]
```diff
@@ -110,4 +107,22 @@ function getLooksDetails($looks, $userid){
     }
 
     return $looks_and_products;
+}
+function getINRPrice ($entity_type_id, $entity_id)
```
Comment (techstunts) [inline backend/istyleyouapi/common.php]: Function is getting called inside a loop?

File: backend/istyleyouapi/latest-looks.php [hunk 2]
```diff
@@ -60,6 +60,8 @@
             $latest_looks_res = mysql_query($latest_looks_sql);
 
             while ($data = mysql_fetch_array($latest_looks_res)) {
+                $price = getINRPrice(2, $data[0]);
```
Comment (techstunts) [inline backend/istyleyouapi/latest-looks.php]: Remove func calling from loop

File: backend/stylists/app/Http/Mapper/LookMapper.php [hunk 3]
```diff
@@ -237,20 +246,64 @@ public function getProducts($look_id)
...
+                $priceData[] = array(
+                    'look_id' => $look_id,
+                    'price_type_id' => $priceType,
+                    'currency_id' => $currency,
+                    'value' => $price,
+                );
+            }
+        }
+
+        $lookPriceExists = LookPrice::select('look_id', 'price_type_id', 'currency_id', 'value')
```
Comment (techstunts) [inline backend/stylists/app/Http/Mapper/LookMapper.php:294]: Migrate old prices to new price structure in Looks

File: backend/stylists/app/Models/Enums/ProductSize.php [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?php
+namespace App\Models\Enums;
+
+
+class ProductSize
+{
+    const NO_ANY = "127";
```
Comment (techstunts) [inline backend/stylists/app/Models/Enums/ProductSize.php]: Change this to 1 and create an entry in lu_sizes table

File: backend/stylists/app/Models/Enums/ProductSize.php [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?php
```
Comment (avinashpatel57) [inline backend/stylists/app/Models/Enums/ProductSize.php:1]: create this entry before going live

**Pull request merged**